nmmo/systems/skill.py

```diff
@@ -265,13 +265,13 @@ def update(self):
     if not config.RESOURCE_SYSTEM_ENABLED:
       return
 
+    if config.IMMORTAL:
+      return
+
     depletion = config.RESOURCE_DEPLETION_RATE
     water = self.entity.resources.water
     water.decrement(depletion)
 
-    if self.config.IMMORTAL:
```

Comment:
i don't love IMMORTAL, maybe we can just get rid of it?

Reply:
IMMORTAL is pretty much for performance testing. Perhaps change to PERFORMANCE_TEST?